package.json

```diff
@@ -60,6 +60,7 @@
     "@spotify/prettier-config": "^9.0.0",
     "command-exists": "^1.2.9",
     "concurrently": "^6.0.0",
+    "eslint-plugin-notice": "^0.9.10",
```

Comment:
Get rid of the `@spotify/eslint-config-oss` now ye? 😅

Reply:
Um do I have to dig through it to see what other things it provides on top of our own cli stuff and copy over that as well? :/

Reply:
There isn't a lot of digging to do, this is all it does: https://github.com/spotify/eslint-config-oss/blob/master/index.js